src/tiler/server.js

```diff
@@ -103,9 +103,11 @@ var config = {
             }
 
             if (body) {
-                s3Obj
-                    .upload({Body: body})
-                    .send(function(err, data) { throw (err); });
```

Comment:
Not sure.  I took that nearly verbatim from the SDK example page for uploading a stream (http://docs.aws.amazon.com/AWSJavaScriptSDK/guide/node-examples.html).  If it works better without it, I can't explain.